utils/build.ps1

```diff
@@ -629,11 +629,13 @@ function Fetch-Dependencies {
     (
         [string]$ZipFileName,
         [string]$BinaryCache,
-        [string]$ExtractPath
+        [string]$ExtractPath,
+        [bool]$CreateExtractPath = $true
     )
 
     $source = Join-Path -Path $BinaryCache -ChildPath $ZipFileName
     $destination = Join-Path -Path $BinaryCache -ChildPath $ExtractPath
+    $extractRoot = if ($CreateExtractPath) { $destination } else { $BinaryCache }
```

Comment:
```suggestion
    $destination = if ($CreateExtractPath) { $destination } else { $BinaryCache }
```

Reply:
@compnerd are you asking for the names `$extractRoot` and `$destination` to be swapped?